### DISCUSSION TITLE
how to improve nodemon start/restart in typescript project even though I setup SWC

### DISCUSSION THREAD

**Discussion by Ram12393** [swc-project/swc#6765] [Q&A]
### Describe the bug

I'm trying to improve nodemon server/start speed.

my script command in `package.json` is

installed `npm install --save-dev ts-node @swc/core @swc/helpers regenerator-runtime`

    scripts: {
    		"dev": "nodemon --watch src server.ts"
    	}
and `tsconfig.json`  is

    {
      "compilerOptions": {
        "baseUrl": ".",
        "target": "ES6",
        "module": "commonjs",
        "removeComments": false,
        "emitDecoratorMetadata": true,
        "experimentalDecorators": true,
        "types": [
          "node",
          "bytebuffer"
        ],
        "moduleResolution": "node",
        "pretty": true,
        "sourceMap": true,
        "skipLibCheck": true,
        "suppressImplicitAnyIndexErrors": true,
        // "noImplicitAny": false,
        "resolveJsonModule": true,
        "outDir": "./build",
        "typeRoots": [
          "./src/interfaces/index.d.ts",
          "node_modules/@types"
        ],
        "paths": {
          "@config/*": [
            "./src/config/*"
          ],
          "@controllers/*": [
            "./src/controllers/*"
          ],
          "@dao/*": [
            "./src/dao/*"
          ],
          "@json/*": [
            "./src/json/*"
          ],
          "@lib/*": [
            "./src/lib/*"
          ],
          "@mappers/*": [
            "./src/mappers/*"
          ],
          "@models/*": [
            "./src/models/*"
          ],
          "@plugins/*": [
            "./src/plugins/*"
          ],
          "@routes/*": [
            "./src/routes/*"
          ],
          "@utils/*": [
            "./src/utils/*"
          ]
        }
      },
      "ts-node": {
        "swc": true
      },
      "filesGlob": [
        "./src/**/*.ts"
      ],
      "exclude": [
        "node_modules",
        "src/lib/customTypings"
      ]
    }
and mainly also mentioned in tsconfig.json file

    ts-node: {
        "swc": true
      }
after Configuring all the steps still `nodemon` server taking 2 min to restart after changes. 

Are there any changes that need to do to the current setup? and I followed this [question][1]

  [1]: https://stackoverflow.com/questions/37979489/how-to-watch-and-reload-ts-node-when-typescript-files-change#70933036

I tried to config SWC still no luck. Is there any way to improve speed

### Expected behavior

After configuration SWC it is not affecting in compile speed

### Version

1.3.25

**Comment by Ram12393** [comment]
May I know why this issue got closed @kdy1

**Reply by kdy1** [reply]
Don't use issue for question

**Reply by Ram12393** [reply]
then how to solve this issue

**Comment by adbutterfield** [comment]
Not sure if it will help you, but I have a repo with TS/nodemon/swc here: https://github.com/adbutterfield/fast-refresh-express
Restart is super fast, but it's also a super tiny repo...
